nominees/clearhealth.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "ClearHealth",
+  "description": "ClearHealth is an open-source practice management (PM) and electronic medical records (EMR/EHR/PHR) system available under the GNU General Public License. It has received attention as a possible open-source option for FQHC and CHC sites.",
+  "website": "http://clear-health.com/",
```

Comment:
@rachita19082 according to the website, it indicates `ClearHealth` has been acquired, not sure what that entails. The DPG standard requires that all projects should be able to document clear ownership of everything the project produces. Unless we are able to verify the clear ownership requirement, we may have to remove this submission and proceed with the others.

Reply:
@nathanbaleeta I have looked into it and I believe the ownership information has changed. I was not able to find clear information about the ownership of the project so I think it will be best to remove it from this submission.

Reply:
Resolved in commit 63b113b.